packages/vscode/package.json

```diff
@@ -19,10 +19,11 @@
     "test": "node ./test/runTest.js"
   },
   "engines": {
-    "vscode": "^1.52.0"
+    "vscode": "^1.56.0"
   },
   "activationEvents": [
-    "onLanguage:astro"
+    "onLanguage:astro",
+    "workspaceContains:astro.config.*"
```

Comment:
Unrelated to the Node 14 changes, I added this so the extension also starts whenever it finds an Astro config file, this is especially convenient for projects who don't have many Astro files